modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -135,3 +135,35 @@ function oe_theme_helper_post_update_20006() {
     $config_factory->getEditable($block)->save();
   }
 }
+
+/**
+ * Add Ratio 3:2 medium image style.
+ */
+function oe_theme_helper_post_update_20007(array &$sandbox) {
```

Comment:
I believe the configuration update needs to take place in ewcms and not in the component.

Reply:
To be confirmed by the coordination team.

Reply:
Discussed with the coordination team, the update path is here since other components might rely on the image style.